test/test_suite_test.go

```diff
@@ -242,23 +240,30 @@ var _ = BeforeAll(func() {
 
 		// Boot / provision VMs if specified by configuration.
 		if config.CiliumTestConfig.Reprovision {
+			var nodesInt int
+			nodes := os.Getenv(k8sNodesEnv)
```

Comment:
Is there a reason this can't be a CLI argument to `ginkgo`? We have a mix of environment variables and CLI flags, which is confusing.

Reply:
Maybe, but I didn't want to change the interface in this PR.

Reply:
just pinging @nebril  in case it wants to give any input...

Reply:
We will move all env vars to CLI flags soon, seems like it's out of scope of this PR.